criu/util.c

```diff
@@ -638,7 +640,7 @@ int cr_system_userns(int in, int out, int err, char *cmd, char *const argv[], un
 		}
 
 		if (WIFEXITED(status)) {
-			if (!(flags & CRS_CAN_FAIL) && WEXITSTATUS(status))
+			if (!(flags & CRS_CAN_ENOENT) && !(flags & CRS_CAN_FAIL) && WEXITSTATUS(status))
```

Comment:
CRS_CAN_ENOENT should not suppress the error message in all cases.

Reply:
Thank you Andrei for the code review! I've updated the pull request to suppress the error message only when `execvp` has exited with `errno == ENOENT` and `CRS_CAN_ENOENT` flag has been set.